src/providers/postgres/qgspostgresprovider.cpp

```diff
@@ -4883,7 +4882,7 @@ QString QgsPostgresProvider::getNextString( const QString &txt, int &i, const QS
     }
     i += match.captured( 1 ).length() + 2;
     jumpSpace( txt, i );
-    if ( !QStringView{txt}.mid( i ).startsWith( sep ) && i < txt.length() )
+    if ( !QStringView{txt} .mid( i ).startsWith( sep ) && i < txt.length() )
```

Comment:
That extra space is out of auto-formatting or a mistake? It shows up in a couple of places here.